crates/workspace/src/pane.rs

```diff
@@ -1650,7 +1652,10 @@ impl Pane {
                                         );
                                     }),
                                 )
-                            });
+                            })
+                            .separator()
+                            .action("Copy Path", Box::new(CopyPath))
+                            .action("Copy Relative Path", Box::new(CopyRelativePath))
```

Comment:
I think I need help, I am not sure what is the way to implement this action in the workspace crate
It looks like each crate must implement actions